src/commands/image/updateImage/updateImage.ts

```diff
@@ -70,4 +70,6 @@ export async function updateImage(context: IActionContext, node?: ContainerAppIt
 
     await wizard.prompt();
     await wizard.execute();
+
+    void showRevisionDraftDeployPopup(context, containerApp);
```

Comment:
Wouldn't it be easier to just include this once in `updateContainerApp`? This should basically only be called wehenver we save the revision draft, right?

Reply:
That's a good idea.  I don't think we'd want it in `updateContainerApp` though because that's the one that does the deploying, but maybe the `RevisionDraftUpdateBaseStep` is a good candidate since it's always used for commands to write to the revision draft.

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/6da78ee2-cb5c-4934-89b4-10e13096512c)

Reply:
Oh yeah, that's the one I mean. I thought `updateContainerApp` was the one that was just creating a draft, but yeah, you get what I mean.

Reply:
I also consolidated this logic in with the base step since it's really just used there now.